scripts/generate_bindings.sh

```diff
@@ -43,9 +43,12 @@ bindgen "$@" \
   -I"$PLAYDATE_C_API" \
   -DTARGET_EXTENSION > "${crankstart_crate_dir}/crankstart-sys/src/bindings_aarch64.rs"
 
+# /usr/include is here because under some versions of Linux (such as Mint 21.1)
+# the arm-non-eabi-gcc -print-sysroot command outputs nothing
 bindgen "$@" \
   -- \
   -I"$PLAYDATE_C_API" \
+  -I"/usr/include" \
```

Comment:
Perhaps we can make it include `$(which arm-non-eabi-gcc)/../include` instead of hard-coding `/usr` for builds which do not print a sysroot?

I'm just generally skeptical of hardcoding stuff like that (and not just because there is no `/usr` on my OS)

Reply:
That is a very reasonable idea.  I have updated the pull request with a new commit.

Reply:
FYI: we have `arm-none-eabi-gcc -print...`, e.g. `arm-none-eabi-gcc -print-search-dirs` or what we're need `arm-none-eabi-gcc -print-sysroot`.
Also `arm-none-eabi-gcc -xc -E -v -` and look after `#include <...> search starts here:`.